docgen/src/main/java/org/apache/servicecomb/toolkit/docgen/ContractsSwaggerUIGenerator.java

```diff
@@ -52,7 +50,7 @@
   @Override
 
   public boolean canProcess(String type) {
-    return DOC_FORMAT.equals(type);
+    return "default".equals(type);
```

Comment:
I don't think default is a validated doc format.

Reply:
DOC_FORMAT was wrong before, canProcess was uesed to choose the default implement by toolkit, it will be supported implement customized by users soon, so no need to fix now.